app/validators/redirect_uri_validator.rb

```diff
@@ -31,4 +33,10 @@ def invalid_ssl_uri?(uri)
     forces_ssl = Doorkeeper.configuration.force_ssl_in_redirect_uri
     forces_ssl && uri.try(:scheme) == 'http'
   end
+
+  def allowed_localhost_uri?(uri)
+    allow_localhost = Doorkeeper.configuration
+                                .allow_localhost_in_redirect_uri
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.